spec/amp-var-substitutions.md

```diff
@@ -65,6 +65,15 @@ For instance:
 ```
 may make a request to something like `https://foo.com/pixel?path=%2Fpage1.html`.
 
+**CLIENT_ID**
+
+Use the special string `CLIENT_ID` to add a per AMP document source origin client identifier.
```

Comment:
do we have a list of the valid values for the required `analytics provider` (first argument)